src/components/AccountSettings.vue

```diff
@@ -37,7 +37,7 @@
 		</AppSettingsSection>
 		<AppSettingsSection :title="t('mail', 'Signature')">
 			<p class="settings-hint">
-				{{ t('mail', 'A signature is added to the text of new messages and replies.') }}
+				{{ t('mail', 'A signature is added to the text of new messages and replies. Select the text to format it.') }}
```

Comment:
I think this line makes it more friendly for people to understand how to format it